.github/workflows/unit_tests.yml

```diff
@@ -1,6 +1,6 @@
 name: unit-tests
 
-on: [push, pull_request]
```

Comment:
was running twice everytime I create a pull request. I verified and this still updates on push